src/Simulation/EntryPointDriver/Azure/Azure.cs

```diff
@@ -0,0 +1,308 @@
+        /// Displays a validation message for a program.
+        /// </summary>
+        /// <param name="message">The validation error message, or null if the program is valid.</param>
+        /// <returns>The exit code.</returns>
+        private static int DisplayValidation(string? message)
+        {
+            if (message is null)
+            {
+                Console.WriteLine("✔️  The program is valid!");
+                return 0;
+            }
+
+            Console.WriteLine("❌  The program is invalid." + Environment.NewLine);
+            Console.WriteLine(message);
+            return 1;
+        }
+
+        /// <summary>
+        /// Displays the job using the output format.
+        /// </summary>
+        /// <param name="job">The job.</param>
+        /// <param name="format">The output format.</param>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown if the output format is invalid.</exception>
+        private static void DisplayJob(IQuantumMachineJob job, OutputFormat format)
+        {
+            switch (format)
+            {
+                case OutputFormat.FriendlyUri:
+                    try
+                    {
+                        Console.WriteLine(job.Uri);
+                    }
+                    catch (Exception ex)
+                    {
+                        DisplayWithColor(
+                            ConsoleColor.Yellow,
+                            Console.Error,
+                            $"The friendly URI for viewing job results could not be obtained.{Environment.NewLine}" +
+                            $"Error details: {ex.Message}{Environment.NewLine}" +
+                            "Showing the job ID instead.");
+
+                        Console.WriteLine(job.Id);
+                    }
+
+                    break;
+                case OutputFormat.Id:
+                    Console.WriteLine(job.Id);
+                    break;
+                default:
+                    throw new ArgumentOutOfRangeException($"Invalid output format '{format}'.");
+            }
+        }
+
+        /// <summary>
+        /// Displays an error to the console.
+        /// </summary>
+        /// <param name="summary">A summary of the error.</param>
+        /// <param name="message">The full error message.</param>
+        private static void DisplayError(string summary, string? message)
+        {
+            DisplayWithColor(ConsoleColor.Red, Console.Error, summary);
+            if (!string.IsNullOrWhiteSpace(message))
+            {
+                Console.Error.WriteLine(Environment.NewLine + message);
+            }
+        }
+
+        /// <summary>
+        /// Logs a message if the verbose setting is enabled.
+        /// </summary>
+        /// <param name="settings">The Azure Quantum submission settings.</param>
+        /// <param name="message">The message.</param>
+        private static void Log(AzureSettings settings, string message)
```

Comment:
Suggest reflecting condition in the name, i.e. LogIfVerbose or similar.

Reply:
Changed name to reflect the condition.